moveit_ros_control_interface/moveit_core_plugins.xml

```diff
@@ -0,0 +1,10 @@
+<class_libraries>
```

Comment:
This file got 3 different names today ;)

At some point I have decided to use a name that reflects the package of the base class, i.e. it corresponds to the tag I have to use in `<export>`.
According to the pluginlib code non-matching base classes are ignored while parsing the XML file, so I could merge both files again.
I have split them before I had a look into the classloader code..

If you insist, I will merge them into one file with that long long name :)